package.json

```diff
@@ -112,7 +113,7 @@
     "hammerjs": "2.0.8",
     "js-yaml": "3.13.1",
     "material-design-icons": "3.0.1",
-    "ng2-ace-editor": "0.3.9",
+    "ng2-ace-editor": "^0.3.9",
```

Comment:
This is not related. Please, revert.

Reply:
i run `npm run build` faild beacuse it ,so i reinstall it. I remember `git add` skip this file, maybe i forget it.

Reply:
So do I need to resubmit pr?

Reply:
No, just revert this change (you can remove the dash manually.

Reply:
👌🏻